### PR TITLE
INT-7660: ingest new device fields

### PR DESCRIPTION
**Pull request by gastonyelmini**
# Description

Adds new ingestion fields.

## Summary

Adds new ingestion fields.

## Type of change

Please leave any irrelevant options unchecked.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [*] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### General Development Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### Integration Development Checklist:

Please leave any irrelevant options unchecked.

- [ ] I have checked for additional permissions required to call any new API
      endpoints, and have documented any additional permissions in
      `jupiterone.md`, where necessary.
- [ ] My changes properly paginate the target service provider's API
- [ ] My changes properly handle rate limiting of the target service provider's
      API
- [ ] My new integration step is instrumented to execute in the correct order
      using `dependsOn`
- [ ] I have referred to the
      [JupiterOne data model](https://github.com/JupiterOne/data-model/tree/main/src/schemas)
      to ensure that any new entities/relationships, and relevant properties,
      match the recommended model for this class of data
- [ ] I have updated the `CHANGELOG.md` file to describe my changes
- [ ] When changes include modifications to existing graph data ingestion, I've
      reviewed all existing managed questions referencing the entities,
      relationships, and their property names, to ensure those questions still
      function with my changes.

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml
Comment (gastonyelmini) [inline .github/workflows/codeql-analysis.yml:13]: This just solves formatting issues required from the prettier config

File: .github/workflows/peril.yml
Comment (gastonyelmini) [inline .github/workflows/peril.yml:1]: This just solves formatting issues required from the prettier config

**Review by Gonzalo-Avalos-Ribas — DISMISSED** [review]
💯

**Review by JuanManuelIbarlucea — DISMISSED** [review]
Looks great!

File: src/types.ts
Comment (ndowmon) [inline src/types.ts]: What is the difference between these two fields? I'm surprised to see `policyStatus` as a string, and `policyStatuses` as numbers.

File: src/types.ts
Comment (ndowmon) [inline src/types.ts:1]: Can you make a separate PR to update the `CODEOWNERS` file to be `@jupiterone/integrations` (instead of `/integrations-staff` and `/integrations-contractors`).

Don't make the change in this PR because it could force @JupiterOne/security to become a required approval.

Screenshot 2023-04-24 at 3 38 46 PM

**Review by ndowmon — DISMISSED** [review]

File: src/types.ts
Comment (gastonyelmini) [reply src/types.ts]: I typed this from the actual response, this is how it looks like: 
Screen Shot 2023-04-24 at 16 40 24

Comment (ndowmon) [reply src/types.ts]: Oh, I see. Should we type like this then?
```suggestion
    policy_status?: string;
    policy_statuses?: { 
      id: number; 
      compliant: boolean; 
    }[];
```

Comment (gastonyelmini) [reply src/types.ts]: sure!

**Review by ndowmon — APPROVED** [review]

**Pull request merged**
